src/js/components/Marathon.jsx

```diff
@@ -513,7 +511,7 @@ var Marathon = React.createClass({
           <div className="container-fluid">
             <div className="navbar-header">
               <a className="navbar-brand" href={window.location.pathname}>
-                <img width="160" height="27" alt="Marathon" src="img/marathon-logo.png" />
+                <img width="160" height="27" alt="Marathon" src="dist/img/marathon-logo.png" />
```

Comment:
I don't think this works when distributing: ![](http://cl.ly/image/1D183S0B3S2N/Image%202015-06-15%20at%2017%3A41%3A16.png)